### ISSUE TITLE
board section on homepage is fixed to a column right now

### ISSUE THREAD

**Issue by yochannah** [kushinauwu/OBF-Homepage-New#34]
no matter how wide the screen, the board section seems to be a bit single-column-ey :)

<img width="1275" alt="image" src="https://user-images.githubusercontent.com/9271438/52747603-26191780-2fdc-11e9-9cfd-7bdc736d19f0.png">

**Comment by yochannah** [comment]
actually I think it's the same on the board page too

**Comment by kushinauwu** [comment]
That seems weird! :o I'm uploading an update with the BOSC abstract submission page, so earlier I checked this as well with different screen sizes, and it seems fine on my local setup right now, so it should look alright in a while with the update! :D
I did replace the bootstrap columns with a CSS grid because the profiles were getting moved up with when a member's name was either smaller or larger. 
 Nomi's picture is still stretched out in the screenshot, but looks fixed after i changed the div dimensions to retain image aspect ratio(at least on the local backup!) So I've got my fingers crossed! I'll let you know when the update is finished :)
